NRWAL/handlers/equations.py

```diff
@@ -37,18 +37,43 @@ def __init__(self, eqn, name=None, default_variables=None):
 
         self._str = None
         self._base_name = name
-        self._eqn = str(eqn)
+        self._eqn_history = [str(eqn)]
```

Comment:
One more thing, can we revert the equation history back to just `self._eqn` now? I dont think these objects need to have histories

Reply:
Sure! This will make the error message less informative, but that's probably fine...